members/SCAU27181_312779.yaml

```diff
@@ -0,0 +1,54 @@
+bioguide: SCAU27181_312779
+contact_form:
+  steps:
+    - visit: "https://lcmspubcontact.lc.ca.gov/PublicLCMS/ContactPopup.php?district=SD30"
+    - fill_in:
+        - name: First Name
+          selector: "#form_name"
+          value: $NAME_FIRST
+          required: true
+        - name: Last Name
+          selector: "#form_lastname"
+          value: $NAME_LAST
+          required: true
+        - name: Address
+          selector: "#form_address"
+          value: $ADDRESS_STREET
+          required: true
+        - name: City
+          selector: "#form_city"
+          value: $ADDRESS_CITY
+          required: true
+        - name: Zip
+          selector: "#form_zip"
+          value: $ADDRESS_ZIP5
+          required: true
+        - name: Phone
+          selector: "#txtPhNumber"
+          value: $PHONE
+          required: false
+        - name: Email
+          selector: "#form_email"
+          value: $EMAIL
+          required: true
+        - name: Message
+          selector: "#comments"
+          value: $MESSAGE
+          options:
+            max_length: 2000
+          required: true
+    - javascript:
+        - name: accent-replace
+          values: ["$NAME_FIRST", "$NAME_LAST"]
+          selectors: ["#form_name", "#form_lastname"]
+          commands: ["for(var i = 0; i < elements.length; i++) { elements[i].value = values[i].replace(/[èēéë]/g,'e').replace(/[àåáä]/g, 'a').replace(/[Á]/g, 'A').replace(/[.&,]/g, '').replace(/[ý]/g, 'y').replace(/[íï]/g, 'i').replace(/[óö]/g, 'o').replace(/[ñ]/g, 'n'); }"]
+    - click_on:
+        - value: Submit
+          selector: "#submitButton"
+    - wait:
+        - value: 1
+    - find:
+        - selector: "tr:nth-child(3)"
+  success:
+    body:
+      excludes: "Select an Issue"
```

Comment:
how confident are you in this exclusion? I think at the very least we should have an `includes` to ensure we aren't on some sort of error page

Reply:
I'm pretty confident. FWIW, this the same "template" we've used for all CA yamls. I haven't seen any indication that we might be getting false positives. 

Of course, it's a different story if say the site is down after clicking `submit`, which would gives us a false positive. 

I'm ok with adding a `includes` with this target's thank you msg, tho, if you think that's safer.

Reply:
generally we should always have an `includes`, even when there is an `excludes` to ensure unexpected failures are guarded against

Reply:
Cool, added it